webpack.config.js

```diff
@@ -0,0 +1,112 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
This file was created with `npx webpack init`, with added plugins to copy icons, html files, and manifest file to the `dist` folder